.github/workflows/build_and_test.yaml

```diff
@@ -47,3 +47,33 @@ jobs:
       run: cd build/debian/cuttlefish_cvdremote && dpkg-buildpackage -i -uc -us -b
     - name: Install package
       run: dpkg -i build/debian/cuttlefish-cvdremote_*_*64.deb || apt-get install -f -y
+
+  build-cuttlefish-cloud-orchestration-x86_64-docker-image:
```

Comment:
About jobs in Github workflow, I'm hesitating moving these jobs into other workflow or not. PLMK if you have any preference.

Reply:
Yes, this workflow should not be here on presubmit, create a new one named artifact running on push only, similar to https://github.com/google/android-cuttlefish/blob/main/.github/workflows/artifacts.yaml.

Reply:
Moved these jobs into another workflow. Those should be run on push only, though now could be run in every pull_request for debugging.